src/app/modules/onboarding/view.nim

```diff
@@ -11,7 +12,7 @@ QtObject:
     View* = ref object of QObject
       delegate: io_interface.AccessInterface
       keycardEvent: KeycardEventDto
-      syncState: int
+      syncState: LocalPairingState
```

Comment:
Right, I think it makes sense to fix the below types to, to prevent future errors like this

Reply:
Yes, but we agreed that we should group other keycard command states, so I decided not to touch them here.

Reply:
Yeah but why can't they use the proper (same) type, instead of a plain `int`?

Reply:
Fair enough, eventhough a separate commit would do for me :)